dictdiffer/__init__.py

```diff
@@ -38,17 +32,47 @@ def diff(first, second, node=None, ignore=None):
         >>> list(result)
         [('change', 'a', ('b', 'c'))]
 
+    PathLimit:
+    list(diff({}, {'a': {'b': 'c'}}))
```

Comment:
can you re-format the docstring?
```python
    """
    Path Limit:

        >>> list(...)
        [...]
    """"
```
   
--
PS: check how does it look like if you compile them locally.